AGPs/AGP-Comprehensive-Aragon-Legal-Review-Program.md

```diff
@@ -0,0 +1,28 @@
+---
+AGP: N/A
+Title: Aragon Comprehensive Legal Review Program
+Author: burrrata (@burrrata)
+Status: Stage IV
+Track: Association
+Created: 2019-10-02
+---
+
+# AGP-X: Aragon Comprehensive Legal Review Program
+
+## Description of desired Association policy change
+Fund up to 3-5 hours of legal council for every Aragon application funded by the Aragon Association via an AGP, the Nest program, or the Flock program. This council would cover the following points:
```

Comment:
```suggestion
Fund up to five hours of legal council for every Aragon application funded by the Aragon Association via an AGP, the Nest program, or the Flock program. This council would cover the following points:
```

Reply:
Since it says "up to", simply putting the upper limit of five hours authorized to be funded removes any ambiguity about whether three or five is the upper limit.

Reply:
Good call. Thanks